STMMC/src/SConscript

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+#
+# Script to build the files found in this directory.
+#
+# Original author Rob Kutschke.
+#
+
+import os, re
+Import('env')
+Import('mu2e_helper')
+
+helper=mu2e_helper(env);
+rootlibs  = env['ROOTLIBS']
+
+mainlib = helper.make_mainlib ( [ 'art_Persistency_Provenance',
+                                  'art_Persistency_Common',
+                                  'art_root_io_TFileService',
+                                  'art_Framework_Services_Registry',
```

Comment:
I have not checked this.  If you don't make histograms in non-module .cc files, you don't need the TFileService in the mainlib link list.